pkg/summary/summarizers.go

```diff
@@ -82,6 +88,18 @@ var (
 		"Failed": true,
 	}
 
+	// For given GVK, This condition Type and this Status, indicates an error or not
+	// e.g.: GVK: helm.cattle.io/v1, HelmChart
+	//            --> Type: JobCreated, Status: True, IsError: false
+	//            --> Type: Failed, Status: True, IsError: false
+	GVKConditionErrorMapping = ConditionTypeStatusErrorMapping{
+		{Group: "helm.cattle.io", Version: "v1", Kind: "HelmChart"}: {
+			{"JobCreated", metav1.ConditionTrue}: false,
+			{"Failed", metav1.ConditionFalse}:    false,
+			{"Failed", metav1.ConditionTrue}:     true,
```

Comment:
It shouldn't be necessary to list the "Failed" type twice here, with "False" => false and "True" => true as the error states. Map each type once, with a list of states that are errors.
```suggestion
			{"JobCreated", nil},
			{"Failed", []string{metav1.ConditionTrue}},
```